src/main/java/task/Task.java

```diff
@@ -0,0 +1,32 @@
+package task;
+
+public class Task {
+    private final String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public static Task parse(String taskInfo) {
+        return new Task(taskInfo);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("[%s] %s", getStatusIcon(), description);
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void unmarkAsNotDone() {
```

Comment:
Maybe change the method name to be markAsNotDone, for better clarity.

Reply:
Thank you for your comment. This issue has been resolved in d5b9beb.